server/config.ts

```diff
@@ -0,0 +1,6 @@
+const config = {
+  audience: '{yourApiIdentifier}',
+  issuerBaseURL: `https://{yourDomain}/`,
```

Comment:
Do we expect customers to update this file? If so, these values should go to environment variables.